tests/unit/type/list.tcl

```diff
@@ -15,14 +15,16 @@ proc write_big_bulk {size} {
     r read
 }
 
-
-# check functionality of plain nodes
+# check functionality compression of plain and zipped nodes
 start_server [list overrides [list save ""] ] {
-
     r config set list-compress-depth 2
     r config set list-max-ziplist-size 1
 
-    # check compression with ziplist nodes
+    # 3 test to check compression with regular ziplist nodes
+    # 1. using push + insert
+    # 2. using push + insert + trim
+    # 3. using push + insert + set
+
     test {reg node check compression with insert and pop} {
```

Comment:
we have here 3 nearly identical tests that are repeated twice.
maybe we can run them inside a foreach loop?
and maybe have it just skip the DEBUG command, or one of the loop iterations if unsupported (checking the tags)?
maybe if it's just these 3 tests, it's not a big deal, but if there are others with similar considerations it may be worth it
i.e. to avoid duplicate code and risk one copy being updated and the other one forgotten.


maybe a similar thing can be done for this:
```
    set str_length 5000000000

    # repeating all the plain nodes basic checks with 5gb values
```